php/php.ini

```diff
@@ -0,0 +1,8 @@
+date.timezone = "Europe/Prague";
+
+memory_limit = 256M
```

Comment:
I would prefer to increase these limits at least to 512MB (maybe rather 1G) for running scripts and 128MB for upload.